terraso_backend/config/settings.py

```diff
@@ -232,6 +232,11 @@
 
 MICROSOFT_CLIENT_ID = config("MICROSOFT_CLIENT_ID", default="")
 MICROSOFT_CLIENT_SECRET = config("MICROSOFT_CLIENT_SECRET", default="")
+MICROSOFT_TENANT = config("MICROSOFT_TENANT", default="")
+MICROSOFT_PRIVATE_KEY = config("MICROSOFT_PRIVATE_KEY", default="").strip()
+MICROSOFT_CERTIFICATE_THUMBPRINT = base64.b64encode(
+    bytes.fromhex(config("MICROSOFT_CERTIFICATE_THUMBPRINT", default="").strip())
+).decode("utf-8")
```

Comment:
question: should this be decoded to `ascii`? It might not matter.

Reply:
Yes, I guess it could be ascii. I generally just treat strings at utf-8 unless specified otherwise, and it works fine here.